runtime/proto/service/log_replication.proto

```diff
@@ -52,4 +53,27 @@ enum LogReplicationEntryType {
   SNAPSHOT_REPLICATED = 4;
   SNAPSHOT_END = 5;
   SNAPSHOT_TRANSFER_COMPLETE = 6;
-}
\ No newline at end of file
+}
+
+enum ReplicationModel {
+  FULL_TABLE = 0;                 // Full table replication (Single Source to Single Sink = 1:1)
```

Comment:
yes it is present in the code under infrastructure/proto/log_replication_metadata.proto, however since the lr client class resides under the runtime directory utilizing that one results in a circular dependency.

Reply:
I see, but then, we can leave it here and remove the other one?

Reply:
+1, I was faced with similar issue, and put it here /infra module could still access it, not vice versa

Reply:
yes, we should be able to migrate all the existing uses of the infrastructure module one if any.